### ISSUE TITLE
UI improvement: statistics dropdown

### ISSUE THREAD

**Issue by nottheend** [ramack/ActivityDiary#276]
In the statistics page, there is a dropdown to select specific period of time base.

It would nice, if the dropdown entries each inrease in height to improve ux

**Comment by ramack** [comment]
yes, this is true. Somehow I don't like this drop down myself. Making it bigger is at least a step towards better UX.
